packages/react/src/atoms/dividers/base.tsx

```diff
@@ -1,29 +1,47 @@
 import React, { ReactNode } from "react";
 import "../../../tailwind.css";
 import { useThemeContext } from "../../theme/ThemeProvider";
-import classNames from "../../utils/classNames";
+import resolvedStyleProps from "../../utils/resolvedStyleProps";
+import LabelBase from "../labels/base";
+import { LabelBaseProps } from "../labels/base";
 
-export interface LabelBaseProps {
-  label: string;
-  htmlFor: string;
-  labelBaseClassNames?: any;
-  labelBaseStyles?: any;
+export interface DividerBaseProps {
+  variant?: string;
+  label?: string;
+  labelBaseProps?: LabelBaseProps;
+  labelAlign?: "left" | "right" | undefined;
+  dividerBaseClasses?: {
```

Comment:
@ShreyDhyani Already dividerBaseClasses suggests that the classes are for dividerBaseClasses, That's why I renamed them.